common/views/components/FilterDrawer/FilterDrawer.js

```diff
@@ -87,7 +87,11 @@ const FilterSection = styled.div.attrs(props => ({
   }
 `;
 
-function FilterDrawer() {
+type Props = {|
+  items: {| title: string, component: Node |}[],
```

Comment:
You could make this more typesafe by having a component `FilterDrawContent` which you could then control with classes / spacing etc?